<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculatorTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QPushButton" name="btnNo9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnNo1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnNoEqual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnDivision">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnNo7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnNoAdd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnNoSub">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="txtDisplay">
        <property name="styleSheet">
         <string notr="true">QLineEdit{
	background-color:green;
	border: 1px solid gray;
	padding: 5px;
	font: 8pt &quot;Fixedsys&quot;;
	color: blue;

}</string>
        </property>
        <property name="text">
         <string>start calc...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnNo6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnNo5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_22">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>M+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnNo8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnNo3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnNo2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnNo0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_24">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>M-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnNo4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_21">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnNoMultiplication">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnNoDecimal">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color:#C0C0C0;
	border: 1px solid gray;
	padding: 5px;

}

QPushButton:pressed{
	background-color:#A9A9A9;
	border: 1px solid gray;
	padding: 5px;

}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
